packages/just-repo-utils/src/normalizePath.ts

```diff
@@ -0,0 +1,9 @@
+import path from 'path';
+
+/**
+ * take a path and make sure it uses forward slashes
+ * @param base - path to put into forward slashed form
+ */
+export function normalizePath(base: string): string {
```

Comment:
I think maybe we should call this normalizeToUnixPath or something to differentiate with path.normalize()?